Apply by doing:
	cd /usr/src
	patch -p0 < 002_openssh2.patch

And then rebuild and install file:
	cd usr.bin/ssh
	make obj
	make cleandir
	make depend
	make
	make install

Index: usr.bin/ssh/channels.c
===================================================================
RCS file: /cvs/src/usr.bin/ssh/channels.c,v
retrieving revision 1.272
retrieving revision 1.272.2.1
diff -u -p -r1.272 -r1.272.2.1
--- channels.c	19 Jan 2008 23:02:40 -0000	1.272
+++ channels.c	3 Apr 2008 03:42:02 -0000	1.272.2.1
@@ -2869,9 +2869,6 @@ x11_create_display_inet(int x11_display_
 			if (bind(sock, ai->ai_addr, ai->ai_addrlen) < 0) {
 				debug2("bind port %d: %.100s", port, strerror(errno));
 				close(sock);
-
-				if (ai->ai_next)
-					continue;
 
 				for (n = 0; n < num_socks; n++) {
 					close(socks[n]);
